Jarvis-Platform/jarvis-studio
# Feature/generic tables to tables

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Use generic components for `tables-to-tables` views


## Todos
### Runs
- [x] Use `listing-component` for `ListingView`
- [x] Use `item-component` for `ItemView`

### Configurations
- [x] Use `listing-component` for `ListingView`
- [x] Use `item-component` for `ItemView`


## Deploy Notes
Review + Merge on master


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/generic-tables-to-tables
```

1. Go to `/data-workflows/tables-to-tables/runs` => Listing should be ok
2. Go to `/data-workflows/tables-to-tables/runs/:id` => Item informations should be ok
3. Go to `/data-workflows/tables-to-tables/configurations` => Listing should be ok
4. Go to `/data-workflows/tables-to-tables/configurations/:id` => Item informations should be ok


## Impacted Areas in Application
* `/data-workflows/tables-to-tables/*` pages
